admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -12,6 +12,8 @@
 <header>
   <nav class="main-header">
 
+    <button class="btn btn-primary-reverse onclick btn-lg unbind ajax-spinner"></button>
```

Comment:
I see. However, it looks like that component was designed with the idea of a button that has a "loading" state in mind. For this kind of loader, we shouldn't create an ad-hoc button, I don't believe it makes sense semantically.

Instead, I think we should use the "dynamic mode" spinner, aka "div with a class". I understand that "dots" spinner shouldn't be used. In that case, we should update the UI Kit to replace the dots with the spinning wheel.

@vincentbz @maximebiloe what do you guys think?